resqpy/grid/grid.py

```diff
@@ -202,7 +202,7 @@ def grid_root(self):
         """Alias for root."""
         return self.root
 
-    def set_modified(self, update_xml=False, update_hdf5=False):
+    def set_modified(self, update_xml = False, update_hdf5 = False):
```

Comment:
having whinged about the addition of an extra space at the start of comments, and removal of the second space before a line continuation, I am actually happy that these spaces are being added here, in accordance with resqpy formatting style!